lightningd/payalgo.c

```diff
@@ -243,7 +243,7 @@ static bool json_pay_try(struct pay *pay)
 	pay->try_parent = tal(pay, char);
 
 	/* Generate random seed */
-	seed = tal_arr(pay->try_parent, u8, ISAAC64_SEED_SZ_MAX);
+	seed = tal_arr(pay->try_parent, u8, sizeof(struct siphash_seed));
```

Comment:
I've produced a patch to actually hand a siphash_seed across the wire, which IMHO makes this neater, but I'll create a separate PR after this.